.github/workflows/build.yml

```diff
@@ -45,9 +45,9 @@ jobs:
         with:
           python-version: "3.x"
           architecture: "x64"
-      - uses: ilammy/msvc-dev-cmd@v1
       - uses: ilammy/setup-nasm@v1
       - name: build-windows-editor-x64
+        shell: powershell
```

Comment:
I thought you needed to use `shell: bash` instead: https://github.com/actions/runner-images/issues/6668#issuecomment-1333430704

The script will have to be adapted to use Bash variable syntax instead of PowerShell's.

Reply:
That's what I tried first but I ran into some linking issues in OpenSSL's horrible perl buildsystem.

The linked issue made it sound like `powershell` defaults to setting `$ErrorActionPreference = "stop"` so I'm trying that. If it doesn't work, we can also likely just call `$ErrorActionPreference = "stop"`.

Reply:
`shell: powershell` does the trick indeed:
https://github.com/godotengine/godot-git-plugin/actions/runs/15262942601/job/42923806465?pr=283